group-generators/helpers/data-providers/index.ts

```diff
@@ -97,6 +100,8 @@ export const dataProvidersInterfacesSchemas: DataProviderInterface[] = [
   attestationStationInterfaceSchema,
   ankrInterfaceSchema,
   degenScoreInterfaceSchema,
+  duneInterfaceSchema,
```

Comment:
can you remove `duneInterfaceSchema` (that's the reason why it cause a problem in the CI)

Reply:
Thanks @MartinGbz, that's done!